util/processinfo.go

```diff
@@ -180,6 +180,8 @@ type SessionManager interface {
 	DeleteInternalSession(se interface{})
 	// GetInternalSessionStartTSList gets all startTS of every transactions running in the current internal sessions.
 	GetInternalSessionStartTSList() []uint64
+	// CheckOldRunningTxn checks if there is an old transaction running in the current sessions
```

Comment:
```suggestion
	// CheckOldRunningTxn checks if there is an old transaction running in the current sessions.
```

BTW the naming is not good..